GoogleForCreators/web-stories-wp
# Ability to expand panels using shortcuts

## Summary

Adds ability to use keyboard shortcuts to expand design panels, implements the <kbd>⌘+k</kbd> shortcut to add a link. 

![ezgif com-optimize](https://user-images.githubusercontent.com/591655/86054130-7399f700-ba0e-11ea-96fa-ba06567c4904.gif)


## Relevant Technical Choices

Extended the `Panel` element with a `shortcut` prop, not sure if that's ideal but it avoids having to use a context.

## User-facing changes

Users are now able to use <kbd>⌘+k</kbd> to add a link to elements.

## Testing Instructions

1. Insert an element that supports links (preferably text, since the text design panel is long enough to observe scrolling)
2. Select the element
3. Press <kbd>⌘+k</kbd>
3. Observe that if the Link design panel was collapsed it would be expanded, scrolled into view and the URL field would be highlighted

---



Fixes #732
